PP_APP/requirements.txt

```diff
@@ -1,43 +1,61 @@
-asgiref==3.2.3
-beautifulsoup4==4.9.0
-cachetools==4.0.0
-certifi==2019.11.28
+absl-py==0.9.0
+asgiref==3.2.7
+astor==0.8.1
+astroid==2.4.0
+cachetools==4.1.0
+certifi==2020.4.5.1
 chardet==3.0.4
-coreapi==2.3.3
-coreschema==0.0.4
-Django==3.0.3
-django-bootstrap4==1.1.1
-django-chartjs==2.0.0
+cycler==0.10.0
+decorator==4.4.2
+Django==3.0.5
+django-cleanup==4.0.0
 django-cors-headers==3.2.1
 djangorestframework==3.11.0
-drf-yasg==1.17.1
+gast==0.3.3
 google-api-core==1.16.0
-google-auth==1.11.2
-google-cloud-vision==0.42.0
+google-auth==1.13.1
+google-cloud-vision==1.0.0
+google-pasta==0.2.0
 googleapis-common-protos==1.51.0
-grpcio==1.27.2
-idna==2.8
-inflection==0.4.0
-itypes==1.1.0
-Jinja2==2.11.2
-MarkupSafe==1.1.1
-packaging==16.8
-Pillow==7.0.0
+grpcio==1.28.1
+h5py==2.10.0
+idna==2.9
+imageio==2.8.0
+isort==4.3.21
+Keras-Applications==1.0.8
+Keras-Preprocessing==1.1.0
+kiwisolver==1.2.0
+lazy-object-proxy==1.4.3
+Markdown==3.2.1
+matplotlib==3.2.1
+mccabe==0.6.1
+networkx==2.4
+numpy==1.18.2
+opencv-python==4.2.0.34
+Pillow==7.1.1
+pkg-resources==0.0.0
 protobuf==3.11.3
 pyasn1==0.4.8
 pyasn1-modules==0.2.8
-pyparsing==2.1.10
+pylint==2.5.0
+pyparsing==2.4.7
+python-dateutil==2.8.1
+python-resize-image==1.1.19
 pytz==2019.3
-requests==2.22.0
+PyWavelets==1.1.1
+requests==2.23.0
 rsa==4.0
-ruamel.yaml==0.16.10
-ruamel.yaml.clib==0.2.0
+scikit-image==0.16.2
+scipy==1.4.1
 six==1.14.0
-soupsieve==2.0
-sqlparse==0.3.0
-uritemplate==3.0.1
+sqlparse==0.3.1
+tensorboard==1.14.0
+tensorflow==1.14.0
+tensorflow-estimator==1.14.0
+tensorflow-serving-api==1.14.0
+termcolor==1.1.0
+toml==0.10.0
+typed-ast==1.4.1
 urllib3==1.25.8
-opencv-python==4.2.0.34
-tensorflow-serving-api==2.1.0
-scikit-image==0.16.2
-python-resize-image==1.1.19
+Werkzeug==1.0.1
+wrapt==1.12.1
```

Comment:
Yeah, everything is tailored towards Linux because our server runs on Ubuntu. What we can do is create another requirements.txt for Mac and push it to master.

Reply:
It runs fine on Mac, except that one issue I mentioned in a different comment.